addons/mail/static/tests/composer/composer.test.js

```diff
@@ -1022,15 +1022,16 @@ test("Canned response can be deleted from the bus", async () => {
     ]);
     await start();
     await openDiscuss(channelId);
-    await contains(".o-mail-Composer-input");
     await insertText(".o-mail-Composer-input", ":");
     await contains(".o-mail-NavigableList-item", { count: 2 });
+    await contains(".o-mail-NavigableList-item", { text: "hello" });
```

Comment:
Note: the test didn't need diff to pass, I'm just making it more robust/maintainable as I had issues with it during dev.